ArduCopter/mode.h

```diff
@@ -1390,6 +1390,11 @@ class ModeZigZag : public Mode {
     // Inherit constructor
     using Mode::Mode;
 
+    enum class position : uint8_t {
```

Comment:
Capital-P for position, please.

Or, as stated below, capital-D-for-Destination.